src/components/album/Album.js

```diff
@@ -53,23 +57,40 @@ function Album({ album, wishListId }) {
         <AddToWishlistButton album={album} wishListId={wishListId} />
       </ButtonGroup>
       {/* Dialog */}
-      <Dialog open={openDialog} onClose={handleCloseDialog}>
+      <Dialog
+        open={openDialog}
+        onClose={handleCloseDialog}
+        fullWidth={fullWidth}
+        maxWidth={maxWidth}
+      >
         <DialogActions>
           <Button onClick={handleCloseDialog} color="primary">
             X
           </Button>
         </DialogActions>
         <DialogContent>
-          <DialogContentText>
+          <DialogContentText
+            style={{
+              display: 'flex',
+              flexDirection: 'column',
+              justifyContent: 'center',
+            }}
+          >
             <div>
               <div>
                 Release Date: {new Date(album.releaseDate).toLocaleDateString()}
               </div>
               <div>Average Rating: {album.averageRating}</div>
             </div>
           </DialogContentText>
-          <AlbumPreview spotifyUrl={album.spotifyUrl} />
         </DialogContent>
+        <AlbumPreview>
```

Comment:
`<AlbumPreview>` component expects a `spotifyUrl`  property to be passed in. Otherwise, it's `useEffect` will fail on line 24 of `AlbumPreview.js` when it tries to do a regex on that property, if that property is undefined.

Reply:
This is where adding propTypes to components can come in handy for helping the developer (and the linter) see what props on a component are required vs optional. But we can circle back to that later :slightly\_smiling\_face:

Reply:
@akosasante go you refert to the use of this prop-types package?  //couldn't implement it without getting errors
![image](https://github.com/Code-the-Dream-School/dd-prac-team4-front/assets/103755090/d1dff2f0-e9a7-46f5-a100-6fb591a10c08)

Reply:
Hmm, maybe pull in the latest from `main` I think we did already implement some prop types in some places: [https://github.com/Code-the-Dream-School/dd-prac-team4-front/blob/945fedf11c6a8694d6d0f7b04dcfcb366f3e2133/src/components/album/Album.js#L80-L83](https://github.com/Code-the-Dream-School/dd-prac-team4-front/blob/945fedf11c6a8694d6d0f7b04dcfcb366f3e2133/src/components/album/Album.js#L80-L83)

And yes we were using the `prop-types` package (lately React officially recommends using that one). You may need to install that in your branch if it's not available.